### ISSUE TITLE
[Desktop Icon Size@lotsrc] Grid layout

### ISSUE THREAD

**Issue by BBric** [linuxmint/cinnamon-spices-applets#100]
Add a grid layout where each icon is moved in the nearest cell (except if it is already occupied), for a manual positioning.

**Comment by NikoKrause** [comment]
You forgot to notify the author of this applet.
You do it like that: @lotsrc

**Comment by jake1164** [comment]
Why is the standard format not consistent across Issues and Pull requests?

IE some titles  are "[uuid] - title" and others "uuid - title"?

**Comment by NikoKrause** [comment]
Because some people can't read.
The Contributing Template is pretty new. We needed to work out some rules first. That's why some of the old issues/pull requests, which have been merged, don't follow these rules, because wie hadn't them yet.
Now that we have them, still some people are ignoring them.
I close issues, if they don't have the applets name in the title at first place.
I don't know if we need to be strict, if the applets uuid is in brackets or not.
